Dockerfile

```diff
@@ -1,14 +1,26 @@
-FROM ubuntu:16.04
+FROM ubuntu:18.04
+
+ENV VPNADDR \
+    VPNUSER \
+    VPNPASS
+
+ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
-  apt-get install -y expect wget net-tools iproute ipppd iptables ssh curl && \
-  rm -rf /var/lib/apt/lists/*
+  apt-get install -y -o APT::Install-Recommends=false -o APT::Install-Suggests=false \
+  expect \
+  net-tools \
+  iproute2 \
+  ipppd \
+  iptables \
+  && apt-get clean -q && apt-get autoremove --purge \
+  && rm -rf /var/lib/apt/lists/*
 
 WORKDIR /root
 
 # Install fortivpn client unofficial .deb
-RUN wget 'https://hadler.me/files/forticlient-sslvpn_4.4.2329-1_amd64.deb' -O forticlient-sslvpn_amd64.deb
-RUN dpkg -x forticlient-sslvpn_amd64.deb /usr/share/forticlient
+COPY forticlient-sslvpn_amd64.deb .
```

Comment:
I actually think of it as a feature that the Dockerfile does not depend on a local `.deb`.
If this is an attempt at removing the version number from the Dockerfile, maybe could use an ENV var for the version though (with the the latest know version as a default value).